ext/standard/streamsfuncs.c

```diff
@@ -788,6 +789,11 @@ PHP_FUNCTION(stream_select)
 
 	PHP_SAFE_MAX_FD(max_fd, max_set_count);
 
+	if (secnull && !usecnull) {
```

Comment:
Maybe we should allow `|| usec == 0` here, so those tests wouldn't have broken? After all, this was documented as the default.

Reply:
I thought about that, too, but there's no reason to pass `0` in these tests.

Reply:
I'm mainly concerned about people who have defined a wrapper around this function that uses `$usec = 0` as the default value, because that's the default we used to document. They would now get an exception.

Reply:
Should we deprecate passing `0` or just allow it?

Reply:
Downgraded passing `0` to a deprecation message.